fieldservice_fleet/hooks.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2019 Brian McMaster
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, api
+
+
+def pre_init_hook(cr):
+    # Check for existing fsm vehicles
+    cr.execute('SELECT * FROM fsm_vehicle')
+    vehicles = []
+    vehicles = cr.dictfetchall()
+    if vehicles:
+        # Add new columns to hold values
+        cr.execute('ALTER TABLE fsm_vehicle ADD fleet_vehicle_id INT;')
+        cr.execute('ALTER TABLE fleet_vehicle ADD is_fsm_vehicle BOOLEAN;')
+
+        # Get a fleet vehicle model to set on the new Fleet vehicle(s)
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        model_id = env['fleet.vehicle.model'].search([], limit=1).id
```

Comment:
Are we assign same model for all vehicle records?

Reply:
Yes. After install, user will need to manually check each new Fleet vehicle and update the model

Reply:
IMO, following two items should be go configuration section: 

After install modules:
---------------------------

- User need to manually check each new Fleet vehicles and update the model as per need.
- Converted vehicles to FSM vehicles have default value `odometer_unit = kilometers`

What do you think?

Reply:
Sounds good to me